docs/geedocs/docsrc/answer/getStarted/overview.rst

```diff
@@ -144,9 +146,8 @@ Overview
       mapping images into a single view. Because the WMS standard is
       used to fetch the images, they can easily be overlaid on one
       another. Supported clients include
-      :doc:`QGIS <../geeServerAdmin/makeWMSRequests>`, ArcGIS/ArcGIS Explorer Desktop
-      , and `Google Earth
-      Pro/EC <https://github.com/google/earthenterprise/wiki/Google-Earth-Enterprise-Client-(EC)>`_.
+      :doc:`QGIS <../geeServerAdmin/makeWMSRequests>`, ArcGIS/ArcGIS
+      Explorer Desktop, and `Google Earth Pro/EC <https://github.com/google/earthenterprise/wiki/Google-Earth-Enterprise-Client-(EC)>`.
```

Comment:
This page only contains the download for EC, not Pro

Reply:
@tst-ccamp : should I remove Google earth Pro from the text
or should I find the link for that include as well ?

Reply:
I think just remove Google Earth Pro. It doesn't work with GEE anyway, so I can't see anyone ever needing the link from the GEE docs